tests/network/vmi_istio.go

```diff
@@ -98,6 +99,10 @@ var istioTests = func(vmType VmType) {
 			{Port: sshPort},
 		}
 	)
+	BeforeEach(func() {
+		checks.SkipTestIfNoFeatureGate(virtconfig.PasstGate)
+	})
```

Comment:
Just the passt tests have to be skipped not all the tests.

Reply:
Right, [fixed](https://github.com/kubevirt/kubevirt/compare/e42feea5cb17dcaabc2142ec54276e720bb7562e..7ae6ccad972d65f004853113d296dafa7071e2a3)

Reply:
Are itsio tests depend on passt?

Reply:
They don't depend on passt, but some of the tests are running on passt VMs.